dream_layer_backend/dream_layer.py

```
@@ -143,23 +154,25 @@ def get_available_models():
     except Exception as e:
         print(f"Error fetching ComfyUI models: {str(e)}")
-    
+
     # Get closed-source models based on available API keys
     try:
         from dream_layer_backend_utils import read_api_keys_from_env
         api_keys = read_api_keys_from_env()
-        
+
         # Append models for each available API key
         for api_key_name, api_key_value in api_keys.items():
```

Ruff (0.11.9)

Loop control variable `api_key_value` not used within loop body

Rename unused `api_key_value` to `_api_key_value`

(B007)